GSoC26_H/hitl/hitl_app.py

```
@@ -0,0 +1,460 @@
+        st.session_state.show_reject = False
+with b3:
+    if st.button("✕  Reject", use_container_width=True):
+        st.session_state.show_reject = True
+        st.session_state.show_modify = False
+
+if st.session_state.get("show_modify"):
+    st.markdown("<br>", unsafe_allow_html=True)
+    st.markdown("**Pick the correct property**")
+    new_prop = st.selectbox(
+        "Correct dbo: property",
+        options=["— type a custom property below —"] + PROPERTY_OPTIONS,
```

Ruff (0.15.18)

[warning] Consider iterable unpacking instead of concatenation

Replace with iterable unpacking

(RUF005)